controller/network/router_messaging.go

```diff
@@ -277,6 +282,12 @@ func (self *RouterMessaging) sendTerminatorValidationRequest(routerId string, up
 		}
 	}
 
+	supportsVerifyV2, err := notifyRouter.VersionInfo.HasMinimumVersion("0.31.1")
```

Comment:
seems like it should be a function with some meta around it like a comment or links to issues or something -- and a constant?